### DISCUSSION TITLE
GIT LFS of  deleted repository

### DISCUSSION THREAD

**Discussion by dharma925** [git-lfs/git-lfs#4731] [Q&A]
I had a github reposity for my unreal engine game, while uploading the files (via lfs) due to some network errors it got stopped in between. Then I unknowingly deleted the repository assuming that the deleting the repository also deletes the LFS. But when I saw the storage of LFS it is showing the storage. Tried all the options available on internet but that storage didn't got cleared. I am running out of my subscription which my college provides. 

So any suggestions to clear the LFS of a deleted repository ?

Thanks  a  lot in advance

**Comment by amirghaz** [comment]
Any solution to this issue?

**Reply by chrisd8088** [reply]
If you're using GitHub as a Git and Git LFS hosting service, and need to delete Git LFS objects, then per their [documentation](https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage#git-lfs-objects-in-your-repository) you will need to delete the repository and all of its forks.

As well, if you want to purge specific objects, you can reach our to GitHub's Support team for assistance.

This project is just for the open-source Git LFS client, and so if you want to contact GitHub and encourage them to develop other options for the deletion of Git LFS objects, you should reach out to their Support team or comment on one of the existing GitHub [community](https://github.com/orgs/community/discussions/128695) [discussions](https://github.com/orgs/community/discussions/4692) on that [topic](https://github.com/orgs/community/discussions/147285).
